.kitchen.yml

```diff
@@ -30,8 +33,6 @@ platforms:
 
 suites:
   - name: "shared_vpc"
-    driver:
-      root_module_directory: test/fixtures/shared_vpc
```

Comment:
Curious - Why is this moved under transport, while the other suites directory are still listed below?

Reply:
Oh good call, let me look at that 🤦
probably should be at the same scope but just under the new path.

Reply:
This should be the right way now

Reply:
Thanks!